drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -4164,7 +4164,7 @@ void RasterizerSceneGLES3::render_scene(const Transform &p_cam_transform, const
 			glBindFramebuffer(GL_READ_FRAMEBUFFER, storage->frame.current_rt->buffers.fbo);
 			glReadBuffer(GL_COLOR_ATTACHMENT0);
 			glBindFramebuffer(GL_DRAW_FRAMEBUFFER, storage->frame.current_rt->fbo);
-			glBlitFramebuffer(0, 0, storage->frame.current_rt->width, storage->frame.current_rt->height, 0, 0, storage->frame.current_rt->width, storage->frame.current_rt->height, GL_DEPTH_BUFFER_BIT, GL_NEAREST);
+			glBlitFramebuffer(0, 0, storage->frame.current_rt->width, storage->frame.current_rt->height, 0, 0, storage->frame.current_rt->width, storage->frame.current_rt->height, GL_COLOR_BUFFER_BIT, GL_NEAREST);
```

Comment:
This code was fine, should not be changed. It needs to resolve the depth buffer in order to use contact shadows

Reply:
> Many apologies, It seems I reviewed the wrong PR. Added comments.

Oops :)

Reply:
> This code was fine, should not be changed. It needs to resolve the depth buffer in order to use contact shadows

I'll test this some more, iOS does not like this, I'm wondering if the glReadBuffer command two lines up is the culprit then, should that be changed to reading from the depth buffer?